Source/Vehicles/Gizmo/Command_TransferToVehicle_Order.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using RimWorld;
+using Verse;
+
+namespace Vehicles
+{
+	public class Command_TransferToVehicle_Order : Command_Target
+	{
+		public static Command_TransferToVehicle_Order Instance { get; }
```

Comment:
Neither does this, the commands should be usable without any singletons

Reply:
I just don't see a reason to create a new instance every time the game calls `Thing.GetGizmos` and i couldn't find a better way to cache it.

Reply:
RimWorld caches it when you first select the thing.  It will be created new anytime you select something else but I don't think that level of optimization is necessary here.  And this might cause problems with multiplayer mods.

Reply:
> RimWorld caches it when you first select the thing.

Honestly, i knew about this but i haven't looked at how exactly the game does this and i've heard that sometimes it doesn't. I've decided to just not rely on implementation details here.

> It will be created new anytime you select something else but I don't think that level of optimization is necessary here.

I do agree that it is a minor optimization. But again, part of the reason i've implemented the gizmos as singletons is because i didn't saw a reason not to (if that makes sense).

> And this might cause problems with multiplayer mods.

Didn't know it could be problematic for the multiplayer mod (never used it).

You can change it if you want to, this PR is editable by maintainers.